distributed/client.py

```diff
@@ -1379,7 +1379,7 @@ def submit(
         func: callable
         *args:
         **kwargs:
-        pure: bool (defaults to True)
+        pure: bool (defaults to opposite of `actor(s)`)
```

Comment:
actors are very rarely used.  I'm disinclined to expect all users to check on that default value in order to understand the default value here.  I'd rather say the factually-wrong-but-almost-always-correct response of just "True" insteead.

Reply:
I'm ok with this.